packages/kotti-ui/source/kotti-filters/KtFilters.vue

```diff
@@ -48,7 +48,7 @@
 import { useTippy } from '@3yourmind/vue-use-tippy'
 import { Yoco } from '@3yourmind/yoco'
 import { computed, defineComponent, ref } from '@vue/composition-api'
-import { roundArrow } from 'tippy.js'
+import { Instance, roundArrow } from 'tippy.js'
```

Comment:
@Juli0GT move this change to https://github.com/3YOURMIND/kotti/pull/689 to keep only data-test related changes in this PR.